packages/webdriverjs/tests/example/package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "axe-core-npm-webdriverjs-example",
+  "name": "@axe-core-npm/webdriverjs-example",
   "private": "true",
```

This might be important: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#private

```suggestion
  "private": true,
```